src/component/mxgraph/BpmnMxGraph.ts

```diff
@@ -30,18 +30,37 @@ export class BpmnMxGraph extends mxGraph {
   }
 
   public customFit(type: FitType): void {
-    let ignoreWidth = false;
-    let ignoreHeight = false;
-    switch (type) {
-      case FitType.Horizontal:
-        ignoreHeight = true;
-        break;
-      case FitType.Vertical:
-        ignoreWidth = true;
-        break;
-    }
+    if (type != FitType.Center) {
+      let ignoreWidth = false;
+      let ignoreHeight = false;
+      switch (type) {
+        case FitType.Horizontal:
+          ignoreHeight = true;
+          break;
+        case FitType.Vertical:
+          ignoreWidth = true;
+          break;
+      }
+
+      this.fit(this.border, false, 0, true, ignoreWidth, ignoreHeight);
+    } else {
+      const margin = 2;
```

Comment:
:warning: for consistency with other fit, the margin should be set to 0.
I am currently working on a feature to make the margin configurable and pass a load option with the fit type. This will be available for all fit type.